### PR TITLE
Update DNS IP regarding multiple IPs

### PR DESCRIPTION
**Pull request by pemontto**
## Proposed change
Clarify change in behaviour now we're returning the lowest IP. Related to https://github.com/home-assistant/core/pull/115218

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/115218
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by c0ffeeca7** [comment]
As this PR is related to a pending code change in core, please point the related docs PR needs to the `next` branch. It is currently targeting the `current` branch. 
image

**Comment by pemontto** [comment]
@c0ffeeca7 have rebased on next and updated target branch :+1:

File: source/_integrations/dnsip.markdown [hunk 1]
```diff
@@ -17,7 +17,7 @@ ha_integration_type: integration
 The **DNS IP** {% term integration %} will expose an IP address, fetched via DNS resolution (every 2 minutes), as its value. It provides both IPv4 and IPv6 lookup as separate sensors depending on accessibility to resolvers.
 
 1. When you enable the {% term integration %}  with the default value, it will query the [OpenDNS](https://www.opendns.com/) nameservers with the hostname `myip.opendns.com`, which will resolve to your external/public IP address.
-2. If you specify a `hostname`, a regular DNS lookup will be performed, providing you the IP the hostname resolves to.
+2. If you specify a `hostname`, a regular DNS lookup will be performed, providing you the IP the hostname resolves to. If your hostname resolves to multiple IP addresses, the lowest IP will returned as the state. Up to the first 10 IPs will be returned in the `IPs` attribute.
```
Comment (c0ffeeca7) [inline source/_integrations/dnsip.markdown]:
```suggestion
2. If you specify a `hostname`, a regular DNS lookup will be performed, providing you the IP the hostname resolves to. If your hostname resolves to multiple IP addresses, the lowest IP will be returned as the state. The first 10 IPs will be returned in ascending order in the `IPs` attribute.
```
Tiny tweak. That's how I understood it from the core PR. Please correct me if I got this wrong.

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
Tiny tweak.

**Review by c0ffeeca7 — DISMISSED** [review]

File: source/_integrations/dnsip.markdown [hunk 2]
```diff
@@ -17,7 +17,7 @@ ha_integration_type: integration
 The **DNS IP** {% term integration %} will expose an IP address, fetched via DNS resolution (every 2 minutes), as its value. It provides both IPv4 and IPv6 lookup as separate sensors depending on accessibility to resolvers.
 
 1. When you enable the {% term integration %}  with the default value, it will query the [OpenDNS](https://www.opendns.com/) nameservers with the hostname `myip.opendns.com`, which will resolve to your external/public IP address.
-2. If you specify a `hostname`, a regular DNS lookup will be performed, providing you the IP the hostname resolves to.
+2. If you specify a `hostname`, a regular DNS lookup will be performed, providing you the IP the hostname resolves to. If your hostname resolves to multiple IP addresses, the lowest IP will be returned as the state. The first 10 IPs will be returned in ascending order in the `IPs` attribute.
```
Comment (gjohansson-ST) [inline source/_integrations/dnsip.markdown]:
```suggestion
2. If you specify a `hostname`, a regular DNS lookup will be performed, providing you the IP the hostname resolves to. If your hostname resolves to multiple IP addresses, the lowest IP will be returned as the state. The first 10 IPs will be returned in ascending order in the `ip_addresses` attribute.
```

**Review by gjohansson-ST — APPROVED** [review]
Approved awaiting parent ✅

**Pull request merged**